chevah/github_hooks_server/server.py

```diff
@@ -48,47 +45,44 @@ def __init__(self, message):
         self.message = message
 
 
-@route('/')
-def root(request):
+def ping(req: func.HttpRequest):
     """
-    Entry point.
+    Simple resource to check that server is up.
     """
-    return ''
+    return func.HttpResponse('pong')
 
 
-@route('/ping',  methods=['GET'])
```

Comment:
The route definitions are now in `serverless.yml`.

Reply:
that is not nice ... but what can we do :)   this is how servless works